### PR TITLE
bump uae4arm

### PR DESCRIPTION
**Pull request by YonKuma**

### CODE REVIEW

**Comment by YonKuma** [comment]
Please don't merge this yet. Just getting a build I can link to

File: packages/games/libretro/uae4arm/package.mk [hunk 1]
```diff
@@ -37,7 +37,11 @@ PKG_TOOLCHAIN="make"
 PKG_AUTORECONF="no"
 
 make_target() {
-  make -f Makefile.libretro platform=unix_aarch64 "CPU_FLAGS=-mcpu=cortex-a35+crypto+crc"
+  if [ "${DEVICE}" = "RG552" ]; then
+    make -f Makefile.libretro platform=rpi4_aarch64
+  else
+    make -f Makefile.libretro platform=rpi3_aarch64
```
Comment (dhwz) [inline packages/games/libretro/uae4arm/package.mk:43]: I think this should be here:
sed -i '/a53/s//a35/' Makefile.libretro

File: packages/games/libretro/uae4arm/package.mk [hunk 2]
```diff
@@ -37,7 +37,11 @@ PKG_TOOLCHAIN="make"
 PKG_AUTORECONF="no"
 
 make_target() {
-  make -f Makefile.libretro platform=unix_aarch64 "CPU_FLAGS=-mcpu=cortex-a35+crypto+crc"
+  if [ "${DEVICE}" = "RG552" ]; then
+    make -f Makefile.libretro platform=rpi4_aarch64
+  else
```
Comment (dhwz) [inline packages/games/libretro/uae4arm/package.mk:42]: I think this should be here:
sed -i '/a53/s//a35/' Makefile.libretro

**Comment by YonKuma** [comment]
@dhwz I manually updated the uae4arm package, since the current version is lacking the rpi4 make instructions. Will that cause any problems with bumping packages?

**Pull request merged**